site/src/components/Footer/Footer.tsx

```diff
@@ -3,13 +3,14 @@ import { makeStyles } from "@material-ui/core/styles"
 import AccountTreeIcon from "@material-ui/icons/AccountTree"
 import AssistantIcon from "@material-ui/icons/Assistant"
 import ChatIcon from "@material-ui/icons/Chat"
+import { colors } from "theme/colors"
 import * as TypesGen from "../../api/typesGenerated"
 
 export const Language = {
   buildInfoText: (buildInfo: TypesGen.BuildInfoResponse): string => {
     return `Coder ${buildInfo.version}`
   },
-  copyrightText: `Copyright \u00a9 ${new Date().getFullYear()} Coder Technologies, Inc. All rights reserved.`,
+  copyrightText: `Copyright \u00a9 ${new Date().getFullYear()} Coder Technologies, Inc.`,
```

Comment:
I removed the "All rights reserved." so the footer width doesn't look bigger than the main container - it looks weird IMO - and also, it is "redundant" since Copyright is about "rights reserved" I guess.